src/playground/with-layout/radio/radio-statuses.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component } from '@angular/core';
+
+interface Option {
```

Comment:
I would prefer to skip this from docs as it complicates the case

Reply:
Ok, removed in 85335606834eeba6d1625fa5f39e446055a3c49b